src/index.js

```diff
@@ -8,6 +8,9 @@ import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
 const store = storeConfig();
+store.subscribe(() => {
+    localStorage.setItem('reduxState', JSON.stringify(store.getState()))
```

Comment:
Currently, localStorage things are not managed by action and reducer.
the idea of this task is to use middleware. In example you can use `redux-thunk`.

This is very dangerous approach in terms of `localStorage`. 
`localStorage` exists only in window env, and it is not supported by 100% browser (please, read about it and please provide me info, what is the `%` of usage currently).
It means, that this app will be totally broken on not supporting browsers (due to `undefined`).
Please, move this to the action (hint-> middleware) and also protect `localStorage` itself.

Reply:
To make this comment full:
- lack of localStorage protection will broke tests
- lack of localStorage will broke server side rendering
as in test env, and on server, there is no window at all.

Reply:
The % of usage localStorage by browser is 93.75%, for today almost all browser support Web Storage (sessionStorage and localStorage) only opera mini and old versions of opera (from 2009), safari (2008) and IE(2006).